### ISSUE TITLE
Spaces in only tasks

### ISSUE THREAD

**Issue by Maxim-Filimonov** [gonzalo-bulnes/dredd-rack#15]
I think there is a bug with space handling for only parameter. It's quite common to have spaces in api names and resources groupes. Usually when dredd needs to be run we would wrap those names in quotes to prevent shell confusion.
For example,
Manually we do
```
dredd ./*.apib ./*.apib *.apib.mb  --only "Mobile Data > Realtime > GET" http://localhost:3000
```
Same parameters does not work with dredd-rack
```
    dredd = Dredd::Rack::Runner.new do |options|
      options.paths_to_blueprints './*.apib ./*.apib *.apib.mb'
      options.only("Mobile Data > Realtime > GET")
    end

=> dredd ./*.apib ./*.apib *.apib.mb  --only Mobile Data > Realtime > GET http://localhost:3000
```
Workaround is to use quotes inside quotes:
```
   options.only("""Mobile Data > Realtime > GET""")
```
OR
```
 options.only("\"Mobile Data > Realtime > GET\"")
```

**Comment by gonzalo-bulnes** [comment]
Yes, yes, yes, that's definitely a bug. I think I prefer escaping the quotes, or maybe use single quotes like this (to preserve the possibility of interpolating variables when building the API names and resources groupes):

```ruby
options.only('"Mobile Data > Realtime > GET\"')
```

What do you think? Do you want to put up a PR?

**Comment by Maxim-Filimonov** [comment]
@gonzalo-bulnes Yeah I can try. I remember there was some way to run command in ruby with passing hash of parameters instead of array. That way it will automatically wrap all parameters in quotes

**Comment by gonzalo-bulnes** [comment]
Hi @Maxim-Filimonov, @Schermy,

I just released **v0.7.1** which fixes this bug. Thank you both for your reports and your help!

**Comment by Schermy** [comment]
@gonzalo-bulnes Thanks so much for fixing this up!
